### DISCUSSION TITLE
Inserir campo assinatura da OS

### DISCUSSION THREAD

**Discussion by AlexJVRodrigues** [RamonSilva20/mapos#2107] [General]
Ola, boa tarde!

Antes de mais nada quero parabenizar pelo sistema é espetacular.

Gostaria de saber se ja existe alguma atualização para a criação de um campo de assinatura da OS?

Desde já agradeço

Att,

Alex

**Comment by cabralwms** [comment]
Se puder testar estou tentando ajudar nesse desenvolvimento. https://github.com/cabralwms/mapos4.40-ass
